cypress/e2e/shared/tasks.test.ts

```diff
@@ -32,35 +32,33 @@ describe('Tasks', () => {
   })
 
   it('can create a task', () => {
-    const taskName = 'Task'
+    const taskName = 'Task A'
     cy.createTaskFromEmpty(taskName, ({name}) => {
       return `import "influxdata/influxdb/v1"
 v1.tagValues(bucket: "${name}", tag: "_field")
 from(bucket: "${name}")
    |> range(start: -2m)`
     })
 
-    cy.getByTestID('task-save-btn').click()
-
+    cy.getByTestID('task-save-btn')
+      .should('not.be.disabled')
+      .click()
     cy.getByTestID('notification-success--dismiss').click()
-
-    cy.getByTestID('task-card')
-      .should('have.length', 1)
-      .and('contain', taskName)
+    cy.getByTestID('task-card--name').contains(taskName)
   })
 
   it('can create a task using http.post', () => {
-    const taskName = 'Task'
+    const taskName = 'Task B'
     cy.createTaskFromEmpty(taskName, () => {
       return `import "http"
 http.post(url: "https://foo.bar/baz", data: bytes(v: "body"))`
     })
 
-    cy.getByTestID('task-save-btn').click()
-
-    cy.getByTestID('task-card')
-      .should('have.length', 1)
-      .and('contain', taskName)
+    cy.getByTestID('task-save-btn')
+      .should('not.be.disabled')
+      .click()
+    cy.getByTestID('notification-success--dismiss').click()
+    cy.getByTestID('task-card--name').contains(taskName)
```

Comment:
if `taskName` is a unique value and there are no other tasks called `Task B` this can be simplified to

```ts
cy.contains(taskName)
```

Reply:
done.